app/cms/models.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+'''
+    Copyright (C) 2018 Gitcoin Core
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+'''
+from wagtail.admin.edit_handlers import FieldPanel, StreamFieldPanel
+from wagtail.core import blocks
+from wagtail.core.fields import RichTextField, StreamField
+from wagtail.core.models import Page
+
+
+class GuideSection(blocks.StructBlock):
+    """Defines the structure for a section in the GuidePage."""
+    heading = blocks.CharBlock()
+    paragraph = blocks.RichTextBlock(required=False)
+    html = blocks.RawHTMLBlock(required=False)
+
+
+class GuideSectionWrapper(blocks.StreamBlock):
+    """A wrapper for GuideSection."""
+    section = GuideSection()
+
+
+class GuidePage(Page):
+    """Defines the structure for the GuidePage."""
```

Comment:
@pinkiebell Generally, looks good.  Going to throw this on staging.

Random note regarding docstrings/spacing with pep-257/258:

Classes call for a blank line like so:

```python
class Class:
    """Some class docstring."""

    var = 'derp'
```

Methods:

```python
def derp():
    """Some derp method."""
    var = 'derp'
```

Some other spacing notes all together:

```python
from __future__ import braces


class Derp:
    """Derp docstring."""

    def derp(self):
        """Some Derp.derp class method docstring."""
        var = 'derp'

    def derp2(self):
        """Some Derp.derp2 class method docstring."""
        var = 'derp2'


def derp(self):
    """Some derp method docstring."""
    var = 'derp'


def derp2(self):
    """Some derp2 method docstring."""
    var = 'derp2'

```

Also, when possible, we follow the google style pep-257/258 docstrings:

```python
def derp(dingo, dance='tango'):
    """Handle processing a dingo based on the provided dance.

    Args:
        dingo (dashboard.Derp): The Derp object.
        dance (str): The dance type.  Defaults to: tango.

    Returns:
        bool: Whether or not the Derp performs the specified dance.

    """
    return dance in dingo.known_dances
```